tests/unit/s2n_self_talk_client_hello_cb_test.c

```diff
@@ -434,6 +479,37 @@ int run_test_reject_handshake_ch_cb(s2n_client_hello_cb_mode cb_mode,
     return S2N_SUCCESS;
 }
 
+int run_test_poll_ch_cb(s2n_client_hello_cb_mode cb_mode,
+    struct s2n_cert_chain_and_key *chain_and_key,
+    struct client_hello_context *ch_ctx, bool cb_poll)
```

Comment:
Why is cb_poll required? Don't you only run this test with cb_poll=true? I thought that was the point.